common/components/Footer/index.jsx

```diff
@@ -1,100 +1,202 @@
-import React, {Component} from 'react';
-import translate, {getTranslators} from 'translations';
-
+import React, { Component } from 'react';
```

Comment:
Most of this is prettier reformats.

The only real change was to use DONATION_ADDRESSES_MAP